src/components/Form/index.js

```diff
@@ -48,19 +48,11 @@ export default class Form extends Component {
       }
 
       findNodesinDOM(props.children, ...formComponentNames)
-      .forEach(node => {
+      .forEach((node) => {
          this.state.data[node.props.name] = {
-<<<<<<< Updated upstream
-            value: node.props.value,
-            meta: {
-               error: null,
-               touched: false
-            }
-=======
             value: node.props.value || '',
```

Comment:
In the case of elements that should have default values as blank strings (such as `Input` and `Textarea`), they should be set on a per-element basis. See relevant code in the aforementioned commit to see what I mean.